packages/web-components/fast-foundation/src/data-grid/data-grid-row.template.ts

```diff
@@ -60,6 +60,7 @@ export function dataGridRowTemplate<T extends FASTDataGridRow>(
             role="row"
             :defaultCellItemTemplate="${cellItemTemplate(options)}"
             :defaultHeaderCellItemTemplate="${headerCellItemTemplate(options)}"
+            aria-selected="${x => (x.selected !== undefined ? x.selected : void 0)}"
```

Comment:
It seems like other Boolean values are not bound the same way. Can this use the `?` operator or should it be a string like the ARIAMixin uses?

Reply:
What's the issue?  This approach renders the attribute correctly as "true"/"false" and not at all if the selected prop is undefined (ie. a non-selectable row).

![image](https://user-images.githubusercontent.com/7649425/218650972-42c53e46-dc01-4fc0-a004-6b8978034049.png)

Reply:
I just noticed it didn't follow the aria property style in the foundation components. I believe that's because this doesn't come from the aria mixin though. It's fine, was just trying to avoid the need for the iif.